### DISCUSSION TITLE
Acceso al Equipo en Team

### DISCUSSION THREAD

**Discussion by lx07975innovaccionmx** [Launch-X-Latam/MisionFrontEnd#161] [General]
Si bien ya solicité el acceso desde el correo con el PASE DE ACCESO ESPACIAL, aun no puedo ver el Equipo en Teams. Y la verdad es que me gustaría ponerme al día en lo que se debe hacer de los temas.

**Comment by doguedogue** [comment]
si se necesita para que veas los videos y sepas que hacer, ponte en contacto con contacto@innovaccion.mx
